aws/vars.tf

```diff
@@ -274,3 +274,9 @@ variable "cloud_provider" {
   description = "the short, lower case form of the cloud provider"
   default     = "aws"
 }
+
+variable "allow_force_destroy" {
+  default     = false
+  type        = bool
+  description = "allows destruction of s3 buckets with objects inside them. should only be enabled for testing tiered storage unless you like data loss :D"
```

Comment:
update this to be something like:

"DANGER: Enabling this option will delete your data in Tiered Storage when terraform destroy is run. Enable this only after careful consideration of the data loss consequences."